website/docs/ThirdParty/WiFiShield/01-Wifi.md

```diff
@@ -2,9 +2,10 @@
 id: WiFiLanding
 title: OpenBCI WiFi
 ---
-## [BUY IT!](https://shop.openbci.com/collections/frontpage/products/wifi-shield)
 
-WiFi Shields are available for sale [on the OpenBCI store](https://shop.openbci.com/collections/frontpage/products/wifi-shield?variant=44534009550).
+::: warning
```

Comment:
Maybe something less harsh than a Warning? Maybe Info?

Reply:
yes, please make it any color you think is appropriate